### PR TITLE
SAM 54 Xplained Pro: fix inverted logic for board led

### PR DESCRIPTION
**Pull request by jgressmann**
This PR fixes the inverted board led logic. the LED was on by default, board_led_write(1) would turn it off.

### CODE REVIEW

File: hw/bsp/same54xplainedpro/same54xplainedpro.c [hunk 1]
```diff
@@ -166,15 +166,16 @@ void board_init(void)
 #endif
 
 	// LED0 init
```
Comment (hathach) [inline hw/bsp/same54xplainedpro/same54xplainedpro.c:168]: Seem like a typo, should be led pin than button

File: hw/bsp/same54xplainedpro/same54xplainedpro.c [hunk 2]
```diff
@@ -166,15 +166,16 @@ void board_init(void)
...
+	gpio_set_pin_function(BUTTON_PIN, GPIO_PIN_FUNCTION_OFF);
 	gpio_set_pin_direction(LED_PIN, GPIO_DIRECTION_OUT);
-	gpio_set_pin_level(LED_PIN, 0);
+	gpio_set_pin_level(LED_PIN, 1);
```
Comment (hathach) [inline hw/bsp/same54xplainedpro/same54xplainedpro.c:171]: How about `board_led_write(0)` to set led to off

**Review by hathach — CHANGES_REQUESTED** [review]
There is led pin typo in function off

File: hw/bsp/same54xplainedpro/same54xplainedpro.c (hunk 1 above)
Comment (jgressmann) [reply hw/bsp/same54xplainedpro/same54xplainedpro.c:168]: Doh!

File: hw/bsp/same54xplainedpro/same54xplainedpro.c (hunk 2 above)
Comment (jgressmann) [reply hw/bsp/same54xplainedpro/same54xplainedpro.c]: Done

**Review by hathach — APPROVED** [review]
Look great, thank you for the pr

**Pull request merged**